catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/model/Search.java

```diff
@@ -275,6 +323,7 @@ private void addObject(Map<String, Object> obj, String name, Object value) {
                 addObject(statusObject, RESULTS, status.getResultCount());
                 addObject(statusObject, HITS, status.getHits());
                 addObject(statusObject, ELAPSED, status.getElapsed());
+                addObject(statusObject, "reasons", status.getReasons());
```

Comment:
I'll probably pull out this `String` literal I'm using as a key into a `public static final String` up top. Any objections?